deeplinks-plugin/deeplinks-plugin.gradle.kts

```diff
@@ -1,6 +1,6 @@
 plugins {
     alias(libs.plugins.fgp.gradle)
-//    alias(libs.plugins.publish)
+    // alias(libs.plugins.fgp.publish)
```

Comment:
Should we remove this line?

Reply:
No, it's just disabled until it is ready